libs/designsystem/src/lib/components/segmented-control/segmented-control.component.ts

```diff
@@ -18,8 +24,20 @@ export class SegmentedControlComponent {
     }
   }
 
-  @HostBinding('class.chip-mode')
-  isChipMode: boolean;
+  @Input() mode: Mode = Mode.default;
+
+  public get isChipMode(): boolean {
```

Comment:
Is this property really public? 
Marked it as such as it was not prefixed with `_`.

Reply:
FWIW, it is not used anywhere in DRB.

Reply:
I will go ahead and remove it 😄

Reply:
https://github.com/kirbydesign/designsystem/pull/1548/commits/32adcab7b1d94097b5daec124e835393b56cfbad